pkgs/development/python-modules/dmt-core/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, fetchPypi, buildPythonPackage, python3Packages, colorama, colormath
+, cycler, h5py, joblib, matplotlib, more-itertools, packaging, pandas, pint
+, pyarrow, pyqtgraph, pyyaml, requests, scikit-rf, scipy, semver, setuptools
+, reprint, verilogae, pytest }:
+
+buildPythonPackage rec {
+  pname = "dmt-core";
+  version = "2.1.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "DMT_core";
+    hash = "sha256-489E+uNn4NgyCwxsUMEPH/1ZuM+5uNq4zx8F88rkHMU=";
+  };
+
+  propagatedBuildInputs = [
+    colorama
+    colormath
+    cycler
+    h5py
+    joblib
+    matplotlib
+    more-itertools
+    packaging
+    pandas
+    pint
+    pyarrow
+    pyqtgraph
+    pyyaml
+    requests
+    scikit-rf
+    scipy
+    semver
+    setuptools
+    reprint
+    verilogae
+  ];
+
+  nativeBuildInputs = [ pytest ];
```

Comment:
Shouldn't this be a `checkInputs`?

Reply:
`nativeCheckInputs` in fact.

Reply:
Yes, I used that initially but got the error 'pytest not installed' So I figured it was okay to enable for the build environment only which was successy.

Reply:
Looks like a direct dependency on pytest, that's a shame. Keeping it in `nativeBuildInputs` is fine I guess. But please file an issue upstream to separate those dependencies and reference it.

https://gitlab.com/dmt-development/dmt-core/-/blob/main/setup.py?ref_type=heads#L57

Reply:
On second thought, use pythonRemoveDeps.